### ISSUE TITLE
Accessibility support

### ISSUE THREAD

**Issue by OlivierMartinez** [vueform/multiselect#179]
### Version

Vue version:  3

### Description

Missing accessibility support:

1.
When using arrows to scroll the results options not readead by the screen reader.

As mentioned here:
https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

2.
All form must have a label.

As mentioned here:
https://www.w3.org/WAI/tutorials/forms/labels/
When the multiselect is searcheable. Theres a input. Would be nice to be able to easily add an id so it's easilly linkable with a label.

**Comment by vincerubinetti** [comment]
In general this library doesn't seem to consider accessibility at all...

If you look at [the examples](https://jsfiddle.net/xf0jzoct/show), there's no `aria-` or `role` attributes or anything like that. Just non-semantic `<div>` elements with tab focus ability.

**Comment by vincerubinetti** [comment]
Related: #22

**Comment by BirdDogJoe** [comment]
You can access the inner input using a ref.

`<Multiselect
      class="border"
      v-model="input"
      mode="tags"
      ref="multiselect" 
    />`

In your TS code just set the input id to whatever you want the id to be. 

`
  mounted() {
    this.$refs.multiselect.input.id = "myId";
  },
`

Now you have the id set and can use that to associate a label to the multiselect box which will allow non-sight users to use the application. Hope this helps!

**Comment by adamberecz** [comment]
Added a11y support in `2.4.0`.
